src/lib/paginator/paginator.html

```diff
@@ -0,0 +1,39 @@
+<div class="mat-paginator-page-length-select">
+  <div class="mat-paginator-items-per-page-label">
+    {{_intl.itemsPerPageLabel}}
+  </div>
+
+  <md-select *ngIf="_displayedPageLengthOptions.length > 1"
+             [ngModel]="pageLength"
+             [aria-label]="_intl.itemsPerPageLabel"
+             (change)="_changePageLength($event.value)">
+    <md-option *ngFor="let pageLengthOption of _displayedPageLengthOptions" [value]="pageLengthOption">
+      {{pageLengthOption}}
+    </md-option>
+  </md-select>
+
+  <div *ngIf="_displayedPageLengthOptions.length <= 1">{{pageLength}}</div>
+</div>
+
+<div class="mat-paginator-range-label">
+  {{_intl.getRangeLabel(currentPageIndex, pageLength, listLength)}}
+</div>
+
+<button md-icon-button
+        class="mat-paginator-navigation-button mat-paginator-navigation-previous"
```

Comment:
Right, good catch thanks. This was there before adding explicit previous/next classes and isn't needed now.